es6/routes/pages.js

```diff
@@ -1,11 +1,60 @@
-import fs   from 'fs';
+import fs from 'fs';
+import { getPosts } from '../blog/wordpressApi';
+import { renderToStaticMarkup } from 'react-dom/server';
+import blogsSection from '../blog/blogsSection';
 
-let pages = {
-  index:  ((req, res) => {
-    fs.readFile(__dirname + '/../../views/index.html', 'utf8', (err, text) => {
-      res.send(text);
-    })
-  })
-}
+// TODO: read and process the template index file at load time.
+// TODO: Use a React Component to process last blog post and default text ("<p>no blogs</p>").
 
-module.exports = pages;
\ No newline at end of file
+export const index
```

Comment:
I was just thinking about this.  You can export many things, but they must have valid JavaScript names.  We couldn't, for instance, export something with the name `blog-single`.  I don't think this will be a problem, especially if I reorganize the file into component-oriented folders.